pywinauto/application.py

```diff
@@ -1135,24 +1156,24 @@ def __getattribute__(self, attr_name):
         return self[attr_name]
 
     def WriteAppData(self, filename):

```

Comment:
I think it's a part of issue #85. I want to review this functionality and make some re-factoring simultaneously.